susemanager-utils/susemanager-sls/modules/roster/uyuni.py

```diff
@@ -89,7 +90,7 @@ def __init__(self, db_config, uyuni_roster_config):
                 )
             )
 
-        log.trace("db_connect string: %s", self.db_connect_str)
+        log.trace("db_connect string: %s", re.sub(r"password='[^']*'", "password='******'", self.db_connect_str))
```

Comment:
Oh, I was wrong in my comment in slack. Instead of using `re` I think it's better to recompose the string with `****** (hidden)` instead of the password value, or show distinct values for `dbname`, `user`, `host` in the separate lines.
I would prefer the last option.